articles/introduction-to-extended-reality/index.md

```diff
@@ -0,0 +1,73 @@
+
+From a user perspective, extended reality works by using all the senses of a person to provide a uniquely immersive experience. These include taste, touch, sight, smell and hearing. It also uses the power of the mind. Various features of extended reality provide users with unique experiences that reconfigures the mind. In most cases, a headset is used to provide such kind of an experience because of its ability to influence all the senses of the user.
+
+The five senses of a person are utilized in immersive technologies as follows;
+- __Sight:__ In extended reality devices, a computer-created reality is seen by users, which gives them an immersive experience. This occurs because the computer-generated reality is laid in a real-world environment. This visual experience is mostly offered in video games, especially on Playstation and Nintendo gaming platforms. In some museums, 360 degrees of cameras have been mounted on XR headsets to provide users with a unique visual experience.
+  
+- __Hearing:__ Sounds in XR devices can be heard from all directions. These devices enable users to perceive sound to be in real situations. In football matches of video games, users will hear the sound of football fans, which happens in real life. 
+  
+- __Touch:__ The sense of touch is integrated into extended reality to improve user experience. In XR devices, gloves and bodysuits are used to give users a sense of touch within the simulation. Wearable devices use haptic technology, which employs a feedback system that helps in the expression of vibrations and motions.
+  
+- __Taste and Smell:__ These are new fields in extended reality that are undergoing research and development. Scent cartridges are used to provide a sense of smell. These cartridges produce pleasant, neutral and unpleasant scents. Electrodes can be placed on a user's tongue to provide a sense of taste.
+
+## Applications of Extended Reality

```

Comment:
Use H3 here, and H4 and H5 for other sub-headings.

Reply:
Done